pkg/controller/daemon/daemon_controller.go

```diff
@@ -206,9 +206,12 @@ func NewDaemonSetsController(
 	dsc.podLister = podInformer.Lister()
 
 	// This custom indexer will index pods based on their NodeName which will decrease the amount of pods we need to get in simulate() call.
-	podInformer.Informer().GetIndexer().AddIndexers(cache.Indexers{
+	err := podInformer.Informer().GetIndexer().AddIndexers(cache.Indexers{
 		"nodeName": indexByPodNodeName,
 	})
+	if err != nil {
+		klog.Infof("adding indexer got %v", err)
```

Comment:
`klog.Errorf` or `klog.Fatalf` is better ? 🤔

Reply:
I chose klog.Warningf
Fatalf would abort the process.